packages/stores/types.ts

```diff
@@ -580,6 +626,11 @@ export type TCoreStores = {
     modules: Record<string, any>;
     notifications: TNotificationStore;
     traders_hub: TTradersHubStore;
+    gtm: Record<string, unknown>;
+    pushwoosh: Record<string, unknown>;
+    contract_replay: Record<string, unknown>;
+    chart_barrier_store: Record<string, unknown>;
+    active_symbols: Record<string, unknown>;
```

Comment:
```suggestion
```
@henry-deriv We should either properly type them or not type them at all, If you are not sure what is the right type just leave it for the next person 🙇🏻